### PR TITLE
Drag main article image from collectionItem

### PR DESCRIPTION
**Pull request by jonathonherbert**
## What's changed?

Following up #742, this PR enables dragging the main article image, regardless of whether it's an override or not.

Depends on #742, so probably best to review and merge that first.

### General
- [ ] 🤖 Relevant tests added
- [x] ✅ CI checks / tests run locally
- [x] 🔍 Checked on CODE

### Client
- [x] 🚫 No obvious console errors on the client (i.e. React dev mode errors)
- [x] 🎛️ No regressions with existing user interactions (i.e. all existing buttons, inputs etc. work)
- [ ] 📷 Screenshots / GIFs of relevant UI changes included

### CODE REVIEW

File: client-v2/src/components/FrontsEdit/CollectionComponents/CollectionItem.tsx [hunk 1]
```diff
@@ -173,11 +173,7 @@ class CollectionItem extends React.Component<ArticleContainerProps> {
       .then(imageData =>
         this.props.addImageToArticleFragment(this.props.uuid, imageData)
       )
-      .catch(err => {
-        // swallowing errors here as the drop may well be an articleFragment
-        // rather than an image which is expected - TBD
-        // console.log('@todo:handle error', err);
-      });
+      .catch(alert);
```
Comment (jonathonherbert) [inline client-v2/src/components/FrontsEdit/CollectionComponents/CollectionItem.tsx]: This is arguably better than swallowing the error -- the comment is no longer valid, as we no longer respond to articleFragment drops here.

File: client-v2/src/shared/components/article/ArticleBodyDefault.tsx [hunk 2]
```diff
@@ -230,7 +230,7 @@ const articleBodyDefault = React.memo(
                 }}
               />
               <ImageMetadataContainer>
-                {imageSlideshowReplace && 'Slidehow'}
+                {imageSlideshowReplace && 'Slideshow'}
```
Comment (jonathonherbert) [inline client-v2/src/shared/components/article/ArticleBodyDefault.tsx]: Couldn't resist 😁

File: client-v2/src/shared/util/__tests__/validateImageSrc.spec.ts [hunk 3]
```diff
@@ -190,7 +190,9 @@ describe('Validate images', () => {
       ).then(
         err => done.fail(err.toString()),
         err => {
-          expect(err.message).toMatch(/unable to locate/i);
+          expect(err.message).toMatch(
+            /There was a problem contacting The Grid/i
```
Comment (jonathonherbert) [inline client-v2/src/shared/util/__tests__/validateImageSrc.spec.ts]: I'm a little more explicit here, I hope that's OK.

File: client-v2/src/shared/components/article/ArticleBodyDefault.tsx (hunk 2 above)
Comment (Reettaphant) [reply client-v2/src/shared/components/article/ArticleBodyDefault.tsx]: Oh dear, thanks for fixing!

File: client-v2/src/shared/util/__tests__/validateImageSrc.spec.ts (hunk 3 above)
Comment (Reettaphant) [reply client-v2/src/shared/util/__tests__/validateImageSrc.spec.ts]: 👍 This is much more useful

**Pull request merged**